src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigurePainter, AreaCalculator {

```

Comment:
Not only Figure can have this behavior
```suggestion
public abstract class Figure implements Drawable, AreaCalculator {
```